pandas/core/strings/object_array.py

```diff
@@ -414,6 +414,33 @@ def _str_lstrip(self, to_strip=None):
     def _str_rstrip(self, to_strip=None):
         return self._str_map(lambda x: x.rstrip(to_strip))
 
+    def _str_removeprefix(self, prefix):
```

Comment:
Can you add type hints for these newly added methods?

Reply:
Just the inputs or return value as well?

Reply:
None of the other methods have return value annotations. What would you like there?

```py
from typing import Any

def _str_removesuffix(self, suffix: str) -> np.ndarray[Any, np.str_]:
```

Or something simpler but not strictly correct?

```py
from typing import Sequence

def _str_removesuffix(self, suffix: str) -> Sequence[str]:
```